### ISSUE TITLE
[AutoComplete] NVDA is not announcing the list items in the popup container when navigating with arrow keys

### ISSUE THREAD

**Issue by zstoyanova9** [telerik/kendo-angular#4263]
**Describe the bug**
NVDA is not announcing the available list items in the popup container of the AutoComplete when navigating with the "Up" and "Down" arrow keys.

**To Reproduce**
1. Open this page - https://www.telerik.com/kendo-angular-ui/components/dropdowns/autocomplete/. 
2. Turn on NVDA screen reader. 
3. Focus the AutoComplete component and insert "C" (or use Alt & Down arrow) to expand the popup list. 
4. Press the "Down" arrow key to navigate through the expanded list. 
5. Observe that NVDA is not announcing the available list items.

**Expected behavior**
NVDA should announce the available list items in the popup container.

**Comment by stnikolova** [comment]
The fix is available in [v15.5.0](https://url.us.m.mimecastprotect.com/s/VRKUC313W3cvBRDYsgQHHG?domain=telerik.com).
